docs/sphinx/theory/theory.rst

```diff
@@ -287,97 +287,6 @@ The anisotropic derivative is used to define the eddy viscosity as
        D_e &= \frac{- (\hat{\partial}_k \widetilde{u}_i) (\hat{\partial}_k \widetilde{\Theta}) \partial_i \widetilde{\Theta} }{(\partial_l \widetilde{\Theta}) (\partial_l \widetilde{\Theta})}
    \end{aligned}
 
-- **Implementation details:**
```

Comment:
This is all removed intentionally?

Reply:
Yup, these were showing the hand unrolled calcs, but those are no longer being used. The tensor form is almost exactly reflected in the code.